seed/seed20.go

```diff
@@ -762,3 +763,30 @@ func (s *seed20) Iter(f func(sn *Snap) error) error {
 	}
 	return nil
 }
+
+func (s *seed20) LoadAutoImportAssertion(commitTo func(*asserts.Batch) error) error {
+	if s.model.Grade() == asserts.ModelDangerous {
+		autoImportAssert := filepath.Join(s.systemDir, "auto-import.assert")
+		af, err := os.Open(autoImportAssert)
+		// ignore if asseertion file does not exists
+		if os.IsNotExist(err) {
+			return nil
+		}
+		if err != nil {
+			logger.Noticef("failed to open auto-import assert(%s): %v\n", autoImportAssert, err)
+			return nil
```

Comment:
Is it intended that this function never returns an error?

Reply:
Yes
Auto import fails here in intentionally converted into the notification (Warning) but boot would continue.
This is to block boot by someone simply adding auto import assertion with the wrong user or signed by the wrong key.
Device boot itself should not have any different results apart of not created linux user. At which point we would run usually console conf....

Reply:
Thanks, can you please write this as a comment in the code, so that it doesn't get lost?

Reply:
Comment added describing why this should not fail.
In addition function does not return error anymore.